config/rbac/role.yaml

```diff
@@ -411,13 +411,8 @@ rules:
   resources:
   - storageclasses
   verbs:
-  - '*'
   - create
   - delete
-  - get
```

Comment:
@OdedViner @iamniting i think we still need  get and list access. can we please check on this one?

Reply:
@Nikhil-Ladha Can you pls help @OdedViner figure that out?

Reply:
Added "get" and "list" verbs based on code and not based manual test 
Get:
https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/util/k8sutil.go#L122

List:
https://github.com/red-hat-storage/ocs-operator/blob/main/pkg/deploy-manager/storagecluster.go#L441

Reply:
`list` verb used here too
https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/storagecluster/backingstorageclasses.go#L30